widgetssdk/src/main/java/com/glia/widgets/core/authentication/AuthenticationManager.kt

```diff
@@ -18,16 +18,6 @@ internal class AuthenticationManager(
         authentication.setBehavior(behavior)
     }
 
-    @Deprecated(
-        "Please use authenticate(String, String?, RequestCallback<Void>)",
-        ReplaceWith("authenticate(jwtToken, null, requestCallback)")
-    )
-    override fun authenticate(requestCallback: RequestCallback<Void>, jwtToken: String) {
-        Logger.logDeprecatedMethodUse(TAG, "authenticate(RequestCallback<Void>, String)")
-        cleanup()
-        authentication.authenticate(jwtToken, null, requestCallback)
-    }
-
```

Comment:
I'd like to ask you to review [this PR](https://github.com/salemove/android-sdk-widgets/pull/1205). 🙂 Once I merge it, you will not need to remove this code in your branch.